sdk/python/feast/stream_feature_view.py

```diff
@@ -304,9 +303,10 @@ def __copy__(self):
             aggregations=self.aggregations,
             mode=self.mode,
             timestamp_field=self.timestamp_field,
-            source=self.source,
+            source=self.stream_source,
```

Comment:
This is a bug in `__copy__`. `self.source` isn't defined for SFV objects; instead, `stream_source` serves the same role as `source`.